qt/scientific_interfaces/EnggDiffraction/EnggVanadiumCorrectionsModel.cpp

```diff
@@ -30,11 +30,7 @@ const std::string EnggVanadiumCorrectionsModel::VANADIUM_INPUT_WORKSPACE_NAME =
     "engggui_vanadium_ws";
 
 std::pair<Mantid::API::ITableWorkspace_sptr, Mantid::API::MatrixWorkspace_sptr>
-EnggVanadiumCorrectionsModel::calculateCorrectionWorkspaces(
-    const std::string &vanadiumRunNumber) const {
-  const auto vanadiumRunName = generateVanadiumRunName(vanadiumRunNumber);
-  const auto vanadiumWS =
```

Comment:
Despite looking like it is unused I think this puts a workspace in the ADS which is then used later in this function.

Reply:
Indeed it does. Nice catch!